scheduler/src/cook/mesos/api.clj

```diff
@@ -1346,22 +1346,18 @@
                                 (mapv #(UUID/fromString %)))
           allow-partial-results (get-in ctx [:request :query-params :partial])
           exists? #(group-exists? (db conn) %)
-          existing-groups (filter exists? requested-guuids)]
-      (if (or (= (count existing-groups) (count requested-guuids))
-              (and allow-partial-results (pos? (count existing-groups))))
-        [true {::guuids existing-groups}]
-        [false {::error (str "UUID "
-                             (str/join
-                               \space
-                               (set/difference (set requested-guuids) (set existing-groups)))
-                             " didn't correspond to a group")}]))
+          {existing-groups true missing-groups false} (group-by exists? requested-guuids)]
+      [false {::allow-partial-results? allow-partial-results
+              ::guuids existing-groups
+              ::bad-uuids missing-groups}])
```

Comment:
Should we use either `guuids` or `uuids` in both of these key names?

Reply:
Renamed `::bad-uuids` → `::non-existing-guuids`.